pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -188,11 +191,17 @@ func (m *manager) Start(activePods ActivePodsFunc, sourcesReady config.SourcesRe
 	}
```

Comment:
You probably want to change this code to returrn an error instead of panicing too.

Reply:
Thanks for noticing.

Reply:
I wrote the patch that recently got merged that introduced this, so wanted to make sure it got covered here on the rebase :)